transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -1,212 +1,640 @@
-from unittest.mock import Mock, call, patch
+from typing import Dict
 
 import pytest
+from django.db.models.expressions import datetime
+from freezegun import freeze_time
 
 from transit_odp.organisation.csv.service_codes import ServiceCodesCSV
 from transit_odp.organisation.factories import DatasetFactory
 from transit_odp.organisation.factories import LicenceFactory as BODSLicenceFactory
 from transit_odp.organisation.factories import (
     OrganisationFactory,
+    SeasonalServiceFactory,
     ServiceCodeExemptionFactory,
     TXCFileAttributesFactory,
 )
 from transit_odp.otc.constants import FLEXIBLE_REG, SCHOOL_OR_WORKS
 from transit_odp.otc.factories import LicenceModelFactory, ServiceModelFactory
 
 pytestmark = pytest.mark.django_db
+CSV_NUMBER_COLUMNS = 17
+
+
+def get_csv_output(csv_string: str) -> Dict[str, list]:
+    result_list = csv_string.replace("\r\n", ",").split(",")[:-1]
+    result = {}
+    len_result_list = len(result_list)
+    start = 0
+    i = 0
+    result.update({"header": result_list[start:CSV_NUMBER_COLUMNS]})
+    start += CSV_NUMBER_COLUMNS
+    for end in range(2 * CSV_NUMBER_COLUMNS, len_result_list + 1, CSV_NUMBER_COLUMNS):
+        result.update({f"row{i}": result_list[start:end]})
+        start = end
+        end += start
+        i += 1
+    return result
```

Comment:
This could trivially be implemented using `csv.reader` but I don't want to waste time rewriting it.

For the record, incrementing `end` is pointless as the value is immediately overwritten by the `range` generator.